src/components/App.jsx

```diff
@@ -20,30 +21,41 @@ import Footer from "./Footer";
 import ErrorBoundary from "./ErrorBoundary";
 import ErrorNotification from "./ErrorNotification";
 
+// FIXME: Currently there seems to be a bug in MUI v5 not allowing theme overriedes
+const theme = createTheme({
+  palette: {
+    secondary: {
+      main: "#e0e0e0",
```

Comment:
It's literally the grey button on the home screen of the app. It's supposed to be a grey color, but mui v5 has a purple color out of the box... I believe I can fix this at the button component level though. I'll delete this custom theme since there seems to be a bug.